DEVELOPER_GUIDE.md

```diff
@@ -556,6 +556,44 @@ Then, you need to apply patterns for git-secrets, you can install the AWS standa
 git secrets --register-aws
 ```
 
+#### Run remote backed storage from dev environment
```

Comment:
Make it consistent with other sections.

```suggestion
#### Remote Backed Storage
```

Since all the examples below are S3, I'd add a subsection for that, giving an opportunity add others later.

```
##### S3
```

Also update TOC on the top of the doc, please.